src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIPacket.cs

```diff
@@ -58,15 +50,11 @@ public string Description
         /// <summary>
         /// Packet validity
         /// </summary>
-        public bool IsInvalid => (_data == null);
+        public bool IsInvalid => _data is null;
```

Comment:
Same as above, the final accessibility will be private but i use public because even internally the distinction between public-internal api surface and private implementation is important to me. Using internal everywhre means you always lose encapsulation.

Reply:
I see